pkg/compiler/compiler.go

```diff
@@ -89,11 +89,10 @@ func (c *Compiler) Compile() error {
 				return core.NewPluginError(p.Name(), err)
 			}
 		}
-		links, err := p.Translate(c.Document.Constructs, c.Document.Resources)
+		_, err := p.Translate(c.Document.Constructs, c.Document.Resources)
```

Comment:
dont we need to change the plugin interface if we remove the links? Am i just missing that?

Reply:
It returns a `core.CloudResourceLink interface` — not the `config.CloudResourceLink struct` that I've rm'ed. That said, it looks like nothing implements that interface, and we always just return `nil`, so I can clean that up too.